src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -72,6 +72,17 @@ type SecretStore =
     | AppService
     | KeyVault of ResourceRef<WebAppConfig>
 
+//
+// I really want to call this a
+// - SiteExtension (annoys me that it collides with type with same name from module SiteExtension) or
+// - SiteExtensionConfiguration (doesn't seem to fit with naming scheme around these parts)
+//
+type Extension = {
```

Comment:
I wonder if you need an explicit type here. Consider an anonymous record e.g. `{| Name : string; Location : Location option |}`; if you remove the Location this becomes even simpler.